README.md

```diff
@@ -0,0 +1,64 @@
+# General registry
+
+The official registry of general Julia packages. This registry of packages is what connects the `Pkg` standard library (see <https://julialang.github.io/Pkg.jl/v1/getting-started/>) to the large ecosystem of Julia packages, allowing them to be installed and updated easily.
+
+Package authors may find the registry's [Guide for package authors](@ref) helpful.
```

Comment:
Change this to a link to that section?

Reply:
Ah, you beat me to it :D

Reply:
I just saw that too-- I just deleted that line, because now the section is just below anyway, so I figure a link isn't needed anymore